tests/reports/test_codelist.py

```diff
@@ -249,7 +249,7 @@ def test_redact_events_table(events_counts_table):
 
 @st.composite
 def distinct_strings_with_common_characters(draw):
-    list_size = draw(st.integers(min_value=10, max_value=100))
+    list_size = draw(st.integers(min_value=3, max_value=20))
```

Comment:
I think this should prevent this test failure in future :crossed_fingers: 
https://github.com/opensafely-core/interactive.opensafely.org/runs/7040459088?check_suite_focus=true

Reply:
Actually, if it doesn't, I've just found that you can setup a CI specific profile. Might be something to consider in future

https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.settings.load_profile